app/test_split_integer.py

```diff
@@ -2,20 +2,41 @@
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    value = 6
+    number_of_parts = 2
+    parts = split_integer(value, number_of_parts)
+
+    assert sum(parts) == value
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    value = 6
+    number_of_parts = 2
+    parts = split_integer(value, number_of_parts)
+
+    assert all(parts == value // number_of_parts for parts in parts),\
```

Comment:
The assertion logic here is incorrect. The expression `all(parts == value // number_of_parts for parts in parts)` is attempting to compare each element in `parts` to the integer division result, but the syntax is flawed. It should be `all(part == value // number_of_parts for part in parts)` to correctly iterate over each element in `parts`.